src/sentry/search/snuba/executors.py

```diff
@@ -132,7 +133,9 @@ def snuba_search(
                 search_filter.key.name == "date"
             ):
                 continue
-            converted_filter = convert_search_filter_to_snuba_query(search_filter)
+            converted_filter = convert_search_filter_to_snuba_query(
+                search_filter, params={"organization_id": organization_id}
```

Comment:
unrelated to this PR, but If possible can we fill out more of the params dict here? Not sure if they're prevented further up, but a decent # of filters depend on the dict

Reply:
Yeah, tbh I don't really love `params` here. It feels like we should maybe just require many of these fields like `organization_id` to be passed down explicitly. Not going to tackle this in this pr though.

I think a lot of the params are specific to discover, so it might not be a huge deal anyway